docs/introduction/dev-environment-quick-start-guide.md

```diff
@@ -1,5 +1,10 @@
 # Dev environment quick start guide
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/https://github.com/openfoodfacts/openfoodfacts-server/)
+> Gitpod will automatically clone and build the repository for you and comes with Docker pre-installed. It is one of the fastest
+> ways to spin up an environment for `openfoodfacts-servbr`, but requires some additional setup to access your development deployment documented
+> in [use-gitpod how-to](../how-to-guides/use-gitpod.md).
+
```

Comment:
Not sure, but I would better add it below as an alternative, because putting it on the top, might confuse someone like if it the way to go, while, this is only an option.

Reply:
agreed, I moved it entirely to the gitpod how-to and replaced with a reference in the quickstart. PTAL